### ISSUE TITLE
Post-reach speed reset

### ISSUE THREAD

**Issue by Ghisallo** [pisi/Reel#250]
[Updated with fix -- see end of post]

I ran into something in my project where after I trigger a reach with a button and then resize my browser, which triggers resizing the div which contains the reel image which is set to responsive -- after doing all that, the reel automagically plays for no apparent reason.  The only way to stop the playback is to click one of the buttons again. That was probably hard to follow, so I made test here for you to see:

http://jsfiddle.net/QGMPB/291/

Test it by:
1. clicking one of the buttons at the bottom
2. resize the result window
3. it spins
4. click a button to stop

I'm not sure if I did something wrong or if I discovered a bug.  I'd be happy with a workaround.  Thanks.

Update:
if I set the speed to 0 at the beginning of the resizeReel function, that stops this unwanted behavior.  I see how it works now.  Reel is loaded when its container is resized and responsive is true.  My reach event triggers set a speed of 1.7.  Therefore, the subsequent load event triggers it to play at a speed of 1.7.  

This works as it should: http://jsfiddle.net/QGMPB/295/

I suggest that reel should change the speed back to the default after completing a reach event to prevent problems like this.  I'm just happy that I found the solution however.

**Comment by pxlwrkz** [comment]
Having the same issue, here are two suggestions for future improvements:
– an event that is triggered after a reach method completes, like "onReachComplete" -- I think this would be very useful anyway. In this case, like Ghisallo points out, we could set the speed to 0
– a boolean option "autoplay"

**Comment by pisi** [comment]
Yes, the issue was exactly that after `reach` was complete, it did not restore back the original speed. Sorry for troubles fellas.

I fixed it to make this happen and I did that by triggering **`reached`** event when reach is complete. Reel automatically restores the speed from `speed` option.

Can you please grab the patched code from [**gh-250-reached-event** branch](/pisi/Reel/tree/gh-250-reached-event) and test it out to verify this solution?

**Comment by pxlwrkz** [comment]
Works great in my test! Thanks!

**Comment by Ghisallo** [comment]
I'm sorry I'm unable to test it for the time being.

**Comment by pisi** [comment]
No problem Ghisallo, I already verified your usecase using your fiddle linking my local JS and it worked perfectly :)

**Comment by Ghisallo** [comment]
That's exactly what I would have done.  Thanks for fixing it.
